server/src/main/scala/net/wiringbits/repositories/daos/DatabaseTablesDAO.scala

```diff
@@ -0,0 +1,249 @@
+package net.wiringbits.repositories.daos
+
+import anorm.{SqlParser, SqlStringInterpolation}
+import net.wiringbits.config.models.DataExplorerSettings
+import net.wiringbits.repositories.models.{Cell, DatabaseTable, TableField, TableMetadata, TableRow}
+import net.wiringbits.util.models.pagination.{Count, Limit, Offset, PaginatedQuery, PaginatedResult}
+
+import java.sql.Connection
+import java.util.UUID
+import scala.collection.mutable.ListBuffer
+
+object DatabaseTablesDAO {
+
+  def all(schema: String = "public")(implicit conn: Connection): List[DatabaseTable] = {
+    SQL"""
+    SELECT table_name
+    FROM information_schema.tables
+    WHERE table_schema = $schema
+      AND table_type = 'BASE TABLE'
+    ORDER BY table_name
+    """.as(tableParser.*)
+  }
+
+  def getTablesInSettings(tableSettings: DataExplorerSettings): List[DatabaseTable] = {
```

Comment:
Unnecesary function, this can be done using a map function 
`    tableSettings.tables.map(x => DatabaseTable(x.tableName))
`